src/app/task-full-descr-teacher/task-full-descr-teacher.component.ts

```diff
@@ -33,16 +33,26 @@ export class TaskFullDescrTeacherComponent {
       'https://sun9-4.userapi.com/c7001/v7001950/4b499/eAKxv9RVmrQ.jpg',
     ]
   };
-
+  deleteStatus: boolean;
   constructor(private activatedRoute: ActivatedRoute, private location: Location, public snackBar: MatSnackBar) {
     const id = this.activatedRoute.snapshot.params.id;
     console.log('id: ' + id);
+    this.deleteStatus = true;
   }
 
   openSnackbar() {
-    this.snackBar.open('Задача удалена', 'Отмена');
-    this.location.back();
-    // setTimeout(this.snackBar.dismiss(), 2000 );
+    const snack = this.snackBar.open(this.deleteStatus ? 'Задача удалена' : 'Задача не удалена',
```

Comment:
Two ternar operators related to one condition smells and isn’t readable.

Construct args array and then spread it in this.snackBar.open